packages/components/src/FormatFile/FormatFile.css

```diff
@@ -32,7 +28,7 @@
   padding: 0;
   border: none;
   text-align: left;
-  background-color: var(--color-transparent);
+  background-color: transparent;
```

Comment:
This token was doing nothing, I figure replacing it is more "safe" but also... maybe it doesn't actually need this declaration.

Reply:
Would rather follow up on that 🤷

Reply:
I think it was _somehow_ getting the correct value???

you're right, that var doesn't exist
![image](https://github.com/GetJobber/atlantis/assets/11843215/65351dd0-7769-4035-b792-de02fc7ebb82)

and yet, the computed value is....

![image](https://github.com/GetJobber/atlantis/assets/11843215/09aa0e4d-75b7-481d-9dbf-91c8f00d4e33)

which is the exact same value we get with your fix!?
![image](https://github.com/GetJobber/atlantis/assets/11843215/3c9a5d8b-6cd4-4390-a503-da32694ad9cd)

I don't know if there's some string matching magic going on where it saw the word "transparent" and ran with it

somehow these changes are equivalent. this is just a much more correct and understandable way of achieving it.